.github/workflows/tests.yml

```diff
@@ -73,22 +73,17 @@ jobs:
         python -m venv env
         source env/bin/activate
         curl -sSL https://install.python-poetry.org | python3 - --version 1.3.2
-    - name: Install dependencies
+    - name: Install dependencies and prepare files
       # always install all -E extras to use a single cache
       run: |
         sudo apt-get install gettext
         source env/bin/activate
         poetry run python -m pip install setuptools -U
         poetry install -E mysql -E pgsql
-        python tools/install_bootstrap.py
-    - name: Prepare files for test run
-      continue-on-error: true
-      run: |
         cp .env.example .env
-        source env/bin/activate
```

Comment:
If it works without sourcing the venv here, does it work in the other steps as well?

Reply:
outdated